apps/condo/domains/banking/gql.js

```diff
@@ -35,7 +35,12 @@ const BankIntegrationOrganizationContext = generateGqlQueries('BankIntegrationOr
 const BANK_TRANSACTION_FIELDS = `{ account { id number } contractorAccount { id name costItem { id name category { id name } } } costItem { id name category { id name } } integrationContext { id enabled } organization { id } number date amount isOutcome currencyCode purpose meta importId importRemoteSystem ${COMMON_FIELDS} }`
 const BankTransaction = generateGqlQueries('BankTransaction', BANK_TRANSACTION_FIELDS)
 
-const BANK_SYNC_TASK_FIELDS = `{ account { id } integrationContext { id } organization { id } property { id } status file { id originalFilename publicUrl mimetype } user { id } totalCount processedCount meta ${COMMON_FIELDS} }`
+const BANK_SYNC_TASK_OPTIONS_FIELDS = [
+    'type',
+    'dateFrom',
+    'dateTo',
+]
+const BANK_SYNC_TASK_FIELDS = `{ account { id } integrationContext { id } organization { id } property { id } status file { id originalFilename publicUrl mimetype } user { id } totalCount processedCount options {${BANK_SYNC_TASK_OPTIONS_FIELDS}} meta ${COMMON_FIELDS} }`
```

Comment:
No need to this complication.
When you interpolate the array in your way, it will produce a string 'one,two,three', but we are not using commas.
```suggestion
const BANK_SYNC_TASK_OPTIONS_FIELDS = 'type dateFrom dateTo'
const BANK_SYNC_TASK_FIELDS = `{ account { id } integrationContext { id } organization { id } property { id } status file { id originalFilename publicUrl mimetype } user { id } totalCount processedCount options {${BANK_SYNC_TASK_OPTIONS_FIELDS}} meta ${COMMON_FIELDS} }`
```

Reply:
Yes, you can. I just followed the example
<img width="1119" alt="image" src="https://user-images.githubusercontent.com/64303474/233285750-917219a6-4460-4ef9-8d51-b13ffb7d3916.png">